cmd/expiration-mailer/main.go

```diff
@@ -286,9 +289,31 @@ func (m *mailer) findExpiringCertificates() error {
 			)
 		}
 		if err != nil {
-			m.log.AuditErr(fmt.Sprintf("expiration-mailer: Error loading certificates: %s", err))
-			return err // fatal
+			m.log.AuditErr(fmt.Sprintf("expiration-mailer: Error loading certificate statuses: %s", err))
+			return err
+		}
+
+		// Now we can sequentially retrieve the certificate details for each of the
+		// certificate status rows
+		var certs []core.Certificate
+		for _, cs := range statuses {
```

Comment:
As the migration for the `JOIN`-less query hasn't been applied yet does this still reduce load? Since the old query still `JOIN`s on the certificate table is ignoring extracting the certificate then and doing it here serially actually faster?